### ISSUE TITLE
running `dwm` through `dbus` breaks `logind` session

### ISSUE THREAD

**Issue by v3natio** [LukeSmithxyz/voidrice#1434]
The following changes
#1402
#1340

break the `logind` session integration for systems using `systemd`.
`logind` already takes care of starting the session with a `dbus` daemon, so there's no need to run one session within another session. This might perhaps be a fix for non-`systemd` init systems.

**Comment by RetroViking** [comment]
Many thanks for posting this, @v3natio . Deleting `dbus-launch` and `dbus-update-activation-environment --all` should fix the duplicate processes on Arch Linux. This [thread](https://bbs.archlinux.org/viewtopic.php?pid=2165005) on the Arch forums explains this in more detail.

Another thing worth mentioning is that the [Arch Wiki](https://wiki.archlinux.org/title/Xinit#xinitrc) recommends adding the last `if` block in `/etc/X11/xinit/xinitrc` to your custom `.xinitrc` profile (for LARBS, that's in `~/.config/x11/xinitrc`). This is the `if` block in question:
```
if [ -d /etc/X11/xinit/xinitrc.d ] ; then
 for f in /etc/X11/xinit/xinitrc.d/?*.sh ; do
  [ -x "$f" ] && . "$f"
 done
 unset f
fi
```
On Arch, this will source a systemd script which is supposed to prevent some problems.

**Comment by v3natio** [comment]
Yup, I saw that as I'm in that thread. The only thing I'm missing is whether that `if` block goes before, or after the `exec dwm` line.

**Comment by RetroViking** [comment]
> The only thing I'm missing is whether that `if` block goes before, or after the `exec dwm` line.

Anywhere before `exec dwm` is fine.
